### PR TITLE
OpenTelemetry Remote Management V2

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request is to add upgrade, edit, and delete information to the OpenTelemetry Remote Management doc.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[Create Beta page for OpenTelemetry Remote Management](https://sumologic.atlassian.net/browse/DOCS-401)

### CODE REVIEW

File: docs/send-data/opentelemetry-collector/remote-management/remote-management-v2.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  <meta name="robots" content="noindex" />
+</head>
+
+<p><a href="/docs/beta"><span className="beta">Beta</span></a></p>
+
+:::info
+This feature is in Beta. To participate, contact your Sumo Logic account executive.
+:::
+
+The Sumo Logic distribution for OpenTelemetry Collector facilitates remote management of data collection configurations, enabling seamless setup from the Sumo Logic UI and deployment to one or more collectors.
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/remote-management/remote-management-v2.md:19]:
```suggestion
The Sumo Logic Distribution for OpenTelemetry Collector facilitates remote management of data collection configurations, enabling seamless setup from the Sumo Logic UI and deployment to one or more collectors.
```

File: docs/send-data/opentelemetry-collector/remote-management/remote-management-v2.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+With remote management, data configuration setup for OTel collectors is done using Source templates. This feature extends our existing [Installed Collector Source](/docs/send-data/installed-collectors/sources) template, allowing attachment to multiple collectors.
+
+Utilize collector tags for grouping collectors, and associate Source templates to these collector groups, reducing redundancy in data collection setup. This process, termed *Collector Linking*, streamlines configuration management.
+
+## Install a Collector and configure the Source Template
+
+### Step 1: Collector installation
+
+Follow the below steps to instal the collector and add uniquely identifiable tags:
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/remote-management/remote-management-v2.md:37]:
```suggestion
Follow the below steps to install the collector and add uniquely identifiable tags:
```

File: docs/send-data/opentelemetry-collector/remote-management/remote-management-v2.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+1. On the **OpenTelemetry Collection** page, click **Add Collector**.
+1. In the **Set up Collector** step, select **Linux** as the platform.<br/><img src={useBaseUrl('img/send-data/linux-install.png')} alt="linux-install" style={{border: '1px solid gray'}} width="800"/>
+1. Enter your **Installation Token**.
+1. Under **Tag data on Collector level**, add a new tag (for example, `application = Apache` as seen in the screenshot below to identify these collectors as having Apache running on them).
+1. For **Collector Settings**, leave them as default (unchecked).
+1. Under **Generate and run the command to install the collector**, copy the command and execute it in your system terminal where the collector needs to be installed.<br/><img src={useBaseUrl('img/send-data/linux-terminal-installation.png')} alt="linux-terminal-installation" width="800"/>
+1. Wait for the installation process to complete, then click **Next** to proceed.
+1. On the next screen, you will see a list of available Source Templates. Select the required Source Template and proceed with the data configuration.
+
+< Add the auto-discovery page screenshot >
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/remote-management/remote-management-v2.md:49]: Commented this out until you have the screenshot ready.

```suggestion
<!-- Add the auto-discovery page screenshot -->
```

File: docs/send-data/opentelemetry-collector/remote-management/remote-management-v2.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+### Step 2: Data configuration
+
+:::info
+A new source template will always be created with the latest verion of the source template.
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/remote-management/remote-management-v2.md:54]:
```suggestion
A new source template will always be created with the latest version of the source template.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
This is so well written, @JV0812 ! Just had a few small tweaks.

File: docs/send-data/opentelemetry-collector/remote-management/remote-management-v2.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+:::note
+You cannot upgrade your source template until all your linked collectors are compatible with the version upgrade.
+:::
+
+1. In the main Sumo Logic menu, select **Manage Data > Collection > Source Template**.
+1. Select the Source Template that you need to upgrade, and click **Upgrade** button.
+1. If required, change the required configuration in the Source Template configuration page, and click **Next**.
+1. Navigate to **Preview Collector(s)** section to view the list of collectors that are compatible and incompatible to the new version of the source template. Follow the either of the below steps if you have incompatible collectors:
+    - Upgrade the collector version to the supported version. Or,
+    - Create a new source template and link the required collectors by collector name and collector tags. Also, delink the collectors added in the new source template to the existing source template.
```
Comment (jpipkin1) [inline docs/send-data/opentelemetry-collector/remote-management/remote-management-v2.md:101]:
```suggestion
    - Create a new source template and link the required collectors by collector name and collector tags. Also, unlink the collectors added in the new source template to the existing source template.
```

**Comment by jpipkin1** [comment]
@JV0812, I removed the hidden Kanso directions with commit "[Fixes for Kanso](https://github.com/SumoLogic/sumologic-documentation/pull/4482/commits/98b0b393c4491e9f23235488e24ff1e93b791854)".

File: docs/send-data/opentelemetry-collector/remote-management/remote-management-v2.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+### Step 1: Collector installation
+
+Follow the below steps to install the collector and add uniquely identifiable tags:
+
+1. [**Classic UI**](/docs/get-started/sumo-logic-ui-classic/). In the main Sumo Logic menu, select **Manage Data > Collection > OpenTelemetry Collection**. <br/>[**New UI**](/docs/get-started/sumo-logic-ui/). In the Sumo Logic top menu select **Configuration**, and then under **Data Collection** select **OpenTelemetry Collection**. You can also click the **Go To...** menu at the top of the screen and select **OpenTelemetry Collection**. 
+1. On the **OpenTelemetry Collection** page, click **Add Collector**.
+1. In the **Set up Collector** step, select **Linux** as the platform.<br/><img src={useBaseUrl('img/send-data/linux-install.png')} alt="linux-install" style={{border: '1px solid gray'}} width="800"/>
+1. Enter your **Installation Token**.
+1. Under **Tag data on Collector level**, add a new tag (for example, `application = Apache` as seen in the screenshot below to identify these collectors as having Apache running on them).
+1. For **Collector Settings**, leave them as default to configure collectore as remotely managed.
```
Comment (jpipkin1) [inline docs/send-data/opentelemetry-collector/remote-management/remote-management-v2.md:44]:
```suggestion
1. For **Collector Settings**, leave them as default to configure collectors as remotely managed.
```

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
This looks really good, @JV0812. I think you should make "source template" and "remote management" lowercase everywhere (including headings) except when documenting screen commands.

**Comment by JV0812** [comment]
@jpipkin1 thanks for pointing on this. I will d that updates.

**Review by jpipkin1 — APPROVED** [review]
Looks good, @JV0812!

**Pull request merged**